### ISSUE TITLE
Remote restart of pellet server

### ISSUE THREAD

**Issue by bdionne** [bdionne/pellet#2]
Only sysadmins at NCI have access to the production servers so we need to be able to remotely restart the pellet server similar to what's done with the protege-server

**Comment by rgrinberg** [comment]
The restart mechanism will be triggered via an http endpoint like protege right?

**Comment by bdionne** [comment]
yes, I'll think we'll want to expose it from the menus also, maybe where we pause the server or in the revision history plugin, so we'll want the functionality in the client api also

**Comment by bdionne** [comment]
This is kind of working, this issue is tied up with #4 - If we do a restart after squashing history and producing a new snapshot, the pellet server files need to be synced up - looks like deleting both `HEAD` and the `reasoner_state.bin` is required
